pkg/vtpm/Dockerfile

```
@@ -47,6 +50,7 @@ RUN ./bootstrap && \
     make -j$(nproc)
 RUN cp lib/.libs/libcommon.so* /out/usr/lib/
 RUN cp tools/.libs/tpm2 /out/usr/bin/
+RUN strip --strip-unneeded /out/usr/lib/*.so*
```

Could we strip the tmp2 bin as well here?
```suggestion
RUN strip --strip-unneeded /out/usr/lib/*.so*
RUN strip --strip-unneeded /out/usr/bin/tpm2
```